src/main/resources/application.properties

```diff
@@ -35,4 +35,12 @@ logging.level.root=info
 
 users.jitsi-technical.password=
 users.jitsi-technical.email=
-users.jitsi-technical.tenantId=0
\ No newline at end of file
+users.jitsi-technical.tenantId=0
+
+weblate.config.server-url=
+weblate.config.api-key=
```

Comment:
how is this solved with the different API-Keys we have?

Reply:
it will be managed on k8s level. So for different instances it will be read from values-secrets-[stage].yaml and mounted into a secret managed by k8s

Reply:
yes, sure. slowly I like the setup :)